notificationworkerlambda/src/main/scala/com/gu/notifications/worker/SenderRequestHandler.scala

```diff
@@ -17,16 +18,37 @@ import org.slf4j.{Logger, LoggerFactory}
 import java.time.{Duration, Instant}
 import scala.jdk.CollectionConverters._
 import scala.concurrent.{ExecutionContext, ExecutionContextExecutor}
+import play.api.libs.json.Json
+import com.amazonaws.services.lambda.runtime.RequestStreamHandler
+import play.api.libs.json.JsValue
+import java.io.InputStream
+import java.io.OutputStream
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder
+import com.gu.notifications.worker.utils.Aws
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.sqs.model.ReceiveMessageRequest
+import com.amazonaws.services.sqs.model.Message
+import com.amazonaws.services.sqs.AmazonSQS
+import scala.annotation.tailrec
+
+
+case class SqsMessage(body: String, sentTimestamp: Long)
+
+object SqsMessage {
+  def fromSqsApiMessage(message: Message) = SqsMessage(message.getBody(), message.getAttributes.getOrDefault("SentTimestamp", "0").toLong)
+
+  def fromSqsEventMessage(message: SQSMessage) = SqsMessage(message.getBody(), message.getAttributes.getOrDefault("SentTimestamp", "0").toLong)
```

Comment:
I want to run the same piece of code on a SQS message no matter whether we use EventBridge or SQS event source, but the classes representing a SQS message are different in `SQSEvent` and in `receiveMesage` API.  So I create a case class to capture relevant fields from SQS messages.